resources/js/src/app/components/item/SingleItem.js

```diff
@@ -9,6 +9,16 @@ Vue.component("single-item", {
 
     computed:
     {
+        isDescriptionTabActive()
+        {
+            return App.config.singleItemData.includes("item.description") && !!this.currentVariation.texts.description.length;
```

Comment:
Du hast hier den Fall nicht bedacht, dass in der Config auch der Wert "all" stehen kann.